bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -1263,6 +1269,20 @@ private void addMergeCommitWarningComment() {
         pr.addComment(message);
     }
 
+    private void addDiffTooLargeWarning() {
+        var existing = findComment(DIFF_TOO_LARGE_WARNING_MARKER);
+        if (existing.isPresent()) {
+            // Only add the comment once per PR
+            return;
+        }
+        var message = "⚠️  @" + pr.author().username() +
+                " This backport pull request is too large for skara bot to get the entire diff from the remote repository. " +
+                "Therefore, skara bot can't evaluate whether this backport is clean or not." +
```

Comment:
The current language style used by the bot does not have it refer to itself as "skara bot".
```suggestion
                " This backport pull request is too large to be automatically evaluated as clean." +
```